docs/design/actions/move.md

```diff
@@ -0,0 +1,29 @@
+# movePod #
+[This project](https://github.com/songbinliu/movePod) demonstrates a method that can move pods, which either are created by ReplicationController, or by ReplicaSet(which may be created by Deployment).
+
+# Method #
+**1.** set the schedulerName of the parent object (ReplicationController, or ReplicaSet) of the pod to a **invalid scheduler**; 
```

Comment:
Make sure the Pod created by RelicaitonController to be Pending until the pod created by the Move operation to be Running.